### PR TITLE
Storybook Hackathon Progress

### PR DESCRIPTION
**Pull request by TurnerRiley**
I took a stab at these refactors/deletions! The refactors all rendered well on the local storybook view, and all deletions seemed like they were in lieu of a unit test so I deleted them (but please check my work if any of these don't seem like they should be deleted or vice versa!).

## Links
Checked off a few items from [this list for EP Tools](https://docs.google.com/spreadsheets/d/1z8r10AcR0v3GimV_-28dJ6QaiaQ3CJoo9yXqa7U_bKs/edit#gid=0).

## PR Checklist:
- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/pd/application_dashboard/cohort_view_table.story.jsx [hunk 1]
```diff
@@ -1,110 +1,108 @@
 import React from 'react';
-import {CohortViewTable} from './cohort_view_table';
-import reactBootstrapStoryDecorator from '../reactBootstrapStoryDecorator';
+import cohort_view_table, {CohortViewTable} from './cohort_view_table';
```
Comment (maddiedierker) [inline apps/src/code-studio/pd/application_dashboard/cohort_view_table.story.jsx]: i think we only want to import the named export (`CohortViewTable`) here, and we can use that for the component field you've got on line 7

**Review by maddiedierker — COMMENTED** [review]
great work, turner! i think we should keep the stories for `UsPhoneNumberInput` -- that looks like a component that we could potentially make more generic and use outside of the `/pd/` space! the other stories you deleted look okay to me to delete, but i'm not very familiar with this area of the codebase, so you probably want some teammates to double check those

File: apps/src/code-studio/pd/application_dashboard/cohort_view_table.story.jsx (hunk 1 above)
Comment (TurnerRiley) [reply apps/src/code-studio/pd/application_dashboard/cohort_view_table.story.jsx]: Sounds good, thanks for looking this over!

**Comment by TurnerRiley** [comment]
> great work, turner! i think we should keep the stories for `UsPhoneNumberInput` -- that looks like a component that we could potentially make more generic and use outside of the `/pd/` space! the other stories you deleted look okay to me to delete, but i'm not very familiar with this area of the codebase, so you probably want some teammates to double check those

Gotcha, I'll add it back in!

File: apps/src/code-studio/pd/form_components/ButtonList.story.jsx [hunk 2]
```diff
@@ -1,66 +1,56 @@
...
-      }
-    ]);
+export default {
+  title: 'ButtonList',
+  component: ButtonList
+};
+
+//
+// TEMPLATE
+//
```
Comment (megcrenshaw) [inline apps/src/code-studio/pd/form_components/ButtonList.story.jsx]: Could you delete these comments? I also see them in `UsPhoneNumberInput.story.jsx`

File: apps/src/code-studio/pd/form_components/UsPhoneNumberInput.story.jsx [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
+
+const Template = args => <UsPhoneNumberInput {...args} />;
+
+//
+// STORIES
+//
+
+export const DefaultPhoneInput = Template.bind({});
+DefaultPhoneInput.args = {
+  label: 'Enter a phone number'
```
Comment (megcrenshaw) [inline apps/src/code-studio/pd/form_components/UsPhoneNumberInput.story.jsx:14]: Love how simple this is!

**Comment by megcrenshaw** [comment]
There is pretty good test coverage for detailViewContents –– I'm good for that one to be deleted.

For WorkshopAssignmentSelect, the component that consumes it (WorkshopAssignmentLoader) has mounted tests which include testing of WorkshopAssignmentSelect. I'm good for that story to be deleted too.

I don't see test coverage for changeLog. It is consumed by detailViewContents, which does have test coverage (mentioned above), but there's nothing in there about the changeLog specifically. We're already mounting detailViewContents for testing, so I could see adding a changeLog test in there (that it shows time, title, and changing_user). I think best case would be to add a unit test in the detailViewContentsTest for changeLog and deleting the storybook file. The lowest lift would be to keep the changeLog story for now (although that's a bit weird because it doesn't work) –– Turner, I'll let you decide which route to take. If you don't add the unit test now, could you make a ticket for it?

File: apps/src/code-studio/pd/form_components/ButtonList.story.jsx (hunk 2 above)
Comment (TurnerRiley) [reply apps/src/code-studio/pd/form_components/ButtonList.story.jsx]: Sounds good!

**Comment by TurnerRiley** [comment]
> There is pretty good test coverage for detailViewContents –– I'm good for that one to be deleted.
> 
> For WorkshopAssignmentSelect, the component that consumes it (WorkshopAssignmentLoader) has mounted tests which include testing of WorkshopAssignmentSelect. I'm good for that story to be deleted too.
> 
> I don't see test coverage for changeLog. It is consumed by detailViewContents, which does have test coverage (mentioned above), but there's nothing in there about the changeLog specifically. We're already mounting detailViewContents for testing, so I could see adding a changeLog test in there (that it shows time, title, and changing_user). I think best case would be to add a unit test in the detailViewContentsTest for changeLog and deleting the storybook file. The lowest lift would be to keep the changeLog story for now (although that's a bit weird because it doesn't work) –– Turner, I'll let you decide which route to take. If you don't add the unit test now, could you make a ticket for it?

Thank you Meg! I added in changelog unit test coverage that checks for roughly what the Storybook was checking (i.e. rendering logs for changing the application status).

**Review by megcrenshaw — APPROVED** [review]
Looks great!

**Review by maddiedierker — APPROVED** [review]

**Pull request merged**